modules/core/src/system.cpp

```diff
@@ -913,6 +913,19 @@ String tempfile( const char* suffix )
     CV_Assert((copied != MAX_PATH) && (copied != (size_t)-1));
     fname = String(aname);
     RoUninitialize();
+#elif _WIN32_WCE
+    LPWSTR temp_dir2[MAX_PATH+1] = {0};
+    LPWSTR temp_file[MAX_PATH+1] = {0};
```

Comment:
Did you test that code?

`LPWSTR` is a pointer.

Reply:
You are correct, I have corrected the code.